backend/src/main/scala/cromwell/backend/standard/callcaching/BlacklistCache.scala

```diff
@@ -18,7 +19,23 @@ case class BlacklistCache(config: CacheConfig) {
       build[String, java.lang.Boolean](falseLoader)
   }
 
-  def isBlacklisted(bucket: String): Boolean = cache.get(bucket)
+  val hitCache = {
+    // Queries to the hit blacklist cache return false by default (i.e. not blacklisted).
+    val falseLoader = new CacheLoader[CallCachingEntryId, java.lang.Boolean]() {
+      override def load(key: CallCachingEntryId): java.lang.Boolean = false
+    }
+
+    CacheBuilder.
+      newBuilder().
+      concurrencyLevel(config.concurrency).
+      maximumSize(config.size).
```

Comment:
Is this `size` likely to be the same size as the number of buckets in the blacklist? Feels like `1000` might not be enough in recent cases, but a blacklist of `1000` buckets is probably more than sufficient?

Reply:
Yes you're right, I just used the same config parameters that were handy but the limit of blacklisted buckets and cache hits probably shouldn't have to be the same.